backend/homework/6/src/main/java/org/example/services/ServiceVehicle.java

```diff
@@ -0,0 +1,34 @@
+package org.example.services;
+
+import jakarta.annotation.PostConstruct;
+import org.example.entities.Vehicle;
+import org.springframework.stereotype.Component;
+
+import java.util.ArrayList;
+import java.util.Comparator;
+import java.util.List;
+
+@Component
+public class ServiceVehicle {
+    private List<Vehicle> vehicles = new ArrayList<>();
+    public Vehicle findMostExpensiveVehicle(){
+        return vehicles.stream()
+                .sorted(Comparator.comparingDouble(Vehicle::getPrice).reversed())
+                .limit(1)
+                .findFirst()
+                .orElse(null);
+    }
+
+    @PostConstruct
+    public void addVehicleToList(){
+        Vehicle vehicle1=new Vehicle(ServiceSpeaker.speaker1(),ServiceTyre.tyre1(),80000+ ServiceSpeaker.speaker1().getPrice()+ServiceTyre.tyre1().getPrice());
```

Comment:
Use meaningful and descriptive names for your variables rather than using numerical suffixes.